toolchain.sh

```diff
@@ -4,6 +4,12 @@
 ## Enter the ps3toolchain directory.
 cd "`dirname $0`" || { echo "ERROR: Could not enter the ps3toolchain directory."; exit 1; }
 
+export PS3DEV_SOURCES=${PWD}/src
```

Comment:
You are assuming that the sciprts are always launched from toolchain.sh.  It used to be possible to run them manually, without using the toolchain.sh wrapper.

Reply:
I can adjust that

Reply:
done